src/components/common/ConnectWallet/__tests__/MPCLogin.test.tsx

```diff
@@ -96,7 +106,30 @@ describe('MPCLogin', () => {
       </MpcWalletProvider>,
     )
 
-    expect(result.getByText('Currently only supported on Gnosis Chain')).toBeInTheDocument()
+    expect(result.getByText('Currently only supported on Goerli')).toBeInTheDocument()
     expect(await result.findByRole('button')).toBeDisabled()
   })
+
+  describe('getSupportedChains', () => {
+    it('returns chain names where social login is enabled', () => {
+      const mockEthereumChain = { chainId: '1', chainName: 'Ethereum', disabledWallets: ['socialLogin'] } as ChainInfo
+      const mockGnosisChain = { chainId: '100', chainName: 'Gnosis Chain', disabledWallets: ['Coinbase'] } as ChainInfo
+      const mockGoerliChain = { chainId: '5', chainName: 'Goerli', disabledWallets: ['TallyHo'] } as ChainInfo
```

Comment:
Nit: If we leave it empty we test that it also works if no wallets are disabled.

```suggestion
      const mockGoerliChain = { chainId: '5', chainName: 'Goerli', disabledWallets: [] } as ChainInfo
```